packages/flame/lib/src/game/camera.dart

```diff
@@ -203,6 +203,18 @@ class Camera extends Projector {
     return worldCoordinates * zoom;
   }
 
+  /// Takes coordinates in the screen space and returns their counter-part in
+  /// the world space
+  Vector2 screenToWorld(Vector2 screenCoordinates) {
+    return unprojectVector(screenCoordinates);
+  }
+
+  /// Takes coordinates in the world space and returns their counter-part in
+  /// the screen space
+  Vector2 worldToScreen(Vector2 worldCoordinates) {
```

Comment:
These could be a one-liner? Not sure about the rules for it.. Just a suggestion :D

Reply:
Nicer to have all these conform to the same style I think

Reply:
We should document it in our styleguide then :D

Reply:
I just mean the same style as for the `project/unproject` methods, I think this is a bit too specific to have in the styleguide. :)

Reply:
Oh I was talking about the expression methods vs block methods :D